### PR TITLE
change: [M3-7646] - Link Cloud Manager README to new documentation pages

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
Tiny PR to link our README to the new docs

## Changes  🔄
- Update "Getting Started" and "Contributing" links to our new GH documentation pages

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ This repository is home to the Akamai Connected **[Cloud Manager](https://cloud.
 
 ## Developing Locally
 
-To get started running Cloud Manager locally, please see the [_Getting Started_ guide](docs/GETTING_STARTED.md).
+To get started running Cloud Manager locally, please see the [Getting Started guide](https://linode.github.io/manager/GETTING_STARTED.html).
```
Comment (abailly-akamai) [inline README.md:48]: i don't think we need to italicize `Getting Started`?

**Review by bnussman-akamai — APPROVED** [review]
Links work ✅

**Review by mjac0bs — APPROVED** [review]
Confirmed links work - thank you!

Happened to notice that our list of commit types for titling our PRs is no longer showing in list form (an existing issue), just a big paragraph. Maybe this section needs to be updated in the future, though, to make it clear how these map to sections in the changelog. (feat to Added, refactor to Tech Story) 
Screenshot 2024-06-17 at 7 55 28 AM

**Comment by abailly-akamai** [comment]
@mjac0bs good catch - see https://github.com/linode/manager/pull/10587

**Pull request merged**